src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -1,59 +1,107 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import { faTimesCircle } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import classNames from "classnames";
 
-import { useChatSidebarControls } from "hooks/chats/chatSidebar";
+import { Banner } from "types/banner";
+
+import { isDefined } from "utils/types";
+
+import { useShowHide } from "hooks/useShowHide";
 
 import { RenderMarkdown } from "components/organisms/RenderMarkdown";
 
+import { LinkButton } from "components/atoms/LinkButton";
+
 import "./AnnouncementMessage.scss";
 
-type AnnouncementMessageProps = {
-  message?: string;
-};
+export interface AnnouncementMessageProps {
+  banner: Banner | undefined;
+  announcementForUser?: boolean;
```

Comment:
what does that mean? if the announcement is for or not-for user?

Reply:
yep, renamed to `isAnnouncementForUser` in ed799a1